containers/verifier/entrypoint.sh

```diff
@@ -1,3 +0,0 @@
-#!/bin/bash -e
-
-exec /usr/local/bin/su-exec nobody /bin/bash -c "$@"
```

Comment:
Are you removing this because it no longer works with the new image or because you noticed that using `USER nobody` in the `Dockerfile` is enough? If it's the latter, then this should really be a separate commit because it has nothing to do with the image change.

Reply:
This was deleted because new the `blender_verfier` docker image has already `entrypoint.sh`. I investigated [entrypoint](https://github.com/golemfactory/golem/blob/develop/apps/entrypoint.sh) and to works with it, we need add the `LOCAL_USER_ID` environmental variable. We could also use `USER` directive to get the same behavior.